src/Symfony/Bridge/Twig/Translation/TwigExtractor.php

```diff
@@ -78,7 +78,18 @@ protected function extractTemplate(string $template, MessageCatalogue $catalogue
         $this->twig->parse($this->twig->tokenize(new Source($template, '')));
 
         foreach ($visitor->getMessages() as $message) {
-            $catalogue->set(trim($message[0]), $this->prefix.trim($message[0]), $message[1] ?: $this->defaultDomain);
+            $id = trim($message[0]);
+
+            $catalogue->set($id, $this->prefix.trim($message[0]), $message[1] ?: $this->defaultDomain);
+
+            if (!empty($message[2])) {
+                $catalogue->setMetadata($id, ['notes' => [
+                    [
+                        'category' => 'symfony-extractor-variables',
+                        'content' => 'Available variables: '.implode(', ', $message[2]),
```

Comment:
Is it ok to use a hard-coded string here? I thought of using a translated string (e.g. to the destination locale passed to `translator:update`), but it may be an overkill...